### DISCUSSION TITLE
Override Stripe Checkout Session from sending receipt_email

### DISCUSSION THREAD

**Discussion by EsotericCoder** [pay-rails/pay#667] [Q&A] [answered]
Wanted to simplify my app and allow Stripe to handle all purchase receipt/refund emails. I enabled it on the Stripe dashboard and disabled emails in Pay config. Unfortunately Stripe will not automatically send these emails if the API is sent a receipt_email when the checkout session is created. Is it possible to override this from happening?

**Comment by excid3** [comment, marked as answer]
Pay does not include a receipt_email. https://github.com/pay-rails/pay/blob/master/lib/pay/stripe/billable.rb#L212
